index.html

```diff
@@ -0,0 +1,249 @@
+<!doctype html>
```

Comment:
this structure is long-winded and sexy. a true an would build the table dynamically, but the correct error-checking functionality would like you to have it hard-coded.